index.js

```diff
@@ -6,26 +6,1159 @@ module.exports = {
       name: 'googleApiKey',
       label: 'Google API key',
       type: 'string',
-      required: true,
+      required: true
     },
     {
       name: 'placeholder',
       label: 'Placeholder',
       type: 'string',
       help: 'Text to display in the field before someone uses it (e.g., to provide additional directions).'
-    }
-  ],
-  arrangeFields: [
-    {
-      name: 'settings',
-      label: 'Field settings',
-      fields: [
-        'fieldLabel',
-        'fieldName',
-        'required',
-        'googleApiKey',
-        'placeholder'
+    },
+    {
+      name: 'types',
+      label: 'Type',
```

Comment:
"Google Places Autocomplete Type" would be clearer.

Reply:
Done